src/constants/oauth2.py

```diff
@@ -0,0 +1,7 @@
+from enum import Enum
+
+
+class GranType(Enum):
+
+    CLIENT_CREDENTIALS = 'CLIENT_CREDENTIALS'  # Client Credentials
```

Comment:
No need to add these comments since OAuth2 is a known flow

Reply:
done.